src/site/content/en/vitals/index.md

```diff
@@ -72,8 +72,10 @@ for most of your users, a good threshold to measure is the **75th percentile**
 of page loads, segmented across mobile and desktop devices.
 
 Tools that assess Core Web Vitals compliance should consider a page passing if
-it meets the recommended targets at the 75th percentile for all of the above
-three metrics.
+it meets the recommended targets at the 75th percentile for all of the Core Web
+Vitals metrics. If a page does not have data for a particular metric (for
```

Comment:
The verbiage we want to use here could be a little tighter (FID is the only metric where a lack of sufficient data allows you to pass as long as LCP and CLS are passing). We wouldn't consider a page as passing if FID and CLS were present, but say, LCP didn't have enough data.